### PR TITLE
Migrate to GitHub Actions

### PR DESCRIPTION
**Pull request by bdemers**
Fixes #682

### CODE REVIEW

**Comment by lhazlewood** [comment]
Is there a reason why you wanted to leave out JDK 11?

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+          java-version: ${{ matrix.java }}
+          distribution: 'temurin'
+          cache: 'maven'
+      - name: Build
+        run: ${{env.MVN_CMD}} verify
+      - name: Code coverage
+        if:  ${{matrix.java}} == '8'
+        run: ${{env.MVN_CMD}} clean clover:setup test && ${{env.MVN_CMD}} -pl . clover:clover clover:check coveralls:report;
+
+  java-7:
```
Comment (lhazlewood) [inline .github/workflows/ci.yml:32]: What about the `export MAVEN_OPTS="-Dhttps.protocols=TLSv1.2 -Xmx512m -XX:MaxPermSize=128m"` line in the current travis build?  It'll fail without the TLSv1.2 flag, no?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+          java-version: '7'
+          architecture: x64
+          cache: 'maven'
+      - name: "Download and install JCE Unlimited Strength Crypto policies for Oracle JDK 7"
+        run: |
+          curl -q -L -C - https://238dj3282as03k369.s3-us-west-1.amazonaws.com/UnlimitedJCEPolicyJDK7.zip -o $RUNNER_TEMP/UnlimitedJCEPolicyJDK7.zip
+          unzip -oj -d "$JAVA_HOME/jre/lib/security" $RUNNER_TEMP/UnlimitedJCEPolicyJDK7.zip \*/\*.jar
+          rm $RUNNER_TEMP/UnlimitedJCEPolicyJDK7.zip
+      - name: Build
+        run: ${{env.MVN_CMD}} verify
```
Comment (lhazlewood) [inline .github/workflows/ci.yml:55]: Also, to ensure coverage checks and reporting, we need from the travis.yml file somewhere:

https://github.com/jwtk/jjwt/blob/2fb6d6bb9c436a27ba109343b26d2d92ced091cb/.travis.yml#L46-L48

**Review by lhazlewood — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 2 above)
Comment (lhazlewood) [reply .github/workflows/ci.yml:55]: Oops - sorry - just now saw that you had this in the JDK 8 build

File: .github/workflows/ci.yml (hunk 1 above)
Comment (bdemers) [reply .github/workflows/ci.yml:32]: yeah, I missed that line 🤦

**Comment by bdemers** [comment]
No reason to leave out 11, just reducing the number of builds running at once, until I can confirm the are running (and reporting to coveralls)

**Review by lhazlewood — APPROVED** [review]
🚢 it!

**Comment by bdemers** [comment]
Rebased on #682
Thanks, @sullis !!

**Pull request merged**